shop-domain/build.gradle

```diff
@@ -1,9 +1,19 @@
 plugins {
     id 'java-library'
     id 'kotlin'
+    id 'kotlin-kapt'
 }
 
 java {
     sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
please use `VERSION_1_8` in all domain modules

Reply:
fixed